recipes/dydxprotocol-v4-client-py/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "dydxprotocol-v4-client-py" %}
+{% set version = "1.1.31" %}
+
+package:
+  name: dydxprotocol-v4-client-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/dydxprotocol/v4-clients/archive/refs/tags/v4-client-js@{{ version }}.tar.gz
+  sha256: eb6ffc4e53aa0a22f195092f2102fb5233c5c4dda7b11700208a1974d1428297
+  patches:
+    - patches/0001-update-v4-proto-version.patch
+    - patches/0002-use-pyext-over-_ubp.patch
+
+build:
+  number: 0
+  noarch: python
+  script:
+    - cd v4-client-py-v2 && poetry build -f wheel
+
+requirements:
+  host:
+    - dydxprotocol-v4-proto-py ==5.0.6
+    - poetry
+    - python >=3.9
+    - typing-extensions ==4.12
+
+outputs:
+  - name: dydxprotocol-v4-client-py
+    script: install-py.sh  # [unix]
+    script: install-py.bat  # [win]
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - poetry
+        - python >=3.9
+      run:
+        - {{ pin_compatible('dydxprotocol-v4-proto-py') }}
+        - bech32 ==1.2
+        - bip-utils ==2.9.3
+        - ecdsa ==0.19
+        - httpx ==0.27
+        - protobuf >=4.23,<4.25
+        - pycryptodome ==3.20
+        - python >=3.9
+        - websocket-client ==1.7
+    test:
+      source_files:
+        - v4-client-py-v2/pyproject.toml
+        - v4-client-py-v2/tests
+      imports:
+        - dydx_v4_client
+      requires:
+        - pip
+        - poetry
+        - pytest
+        - pytest-asyncio
+        - pytest-dotenv
+      commands:
+        - pip check
+        - cd v4-client-py-v2 && poetry run pytest || true
+
+  # - Naming conversion
+  - name: dydxprotocol_v4_client_py
+    requirements:
+      run:
+        - {{ pin_subpackage('dydxprotocol-v4-client-py', exact=True) }}
+    test:
+      imports:
+        - dydx_v4_client
+      requires:
+        - pip
+      commands:
+        - pip check
+
+  # Package name required by v4-client upstream pyproject.toml
+  - name: v4-client
```

Comment:
This won't be visible in any way to the `pyproject.toml`. This is a conda package and not showing up in `pip`/`python` metadata. Please remove this entry.

Reply:
I don't understand, would it not be installable, like `conda install v4-client`? My comment is likely misleading, I menat to say that if a package has a `v4-client` requirement in its pyprokect.toml (or else), then a user would be able to `conda search v4-client` and add it to its `environment.yml`

Reply:
With that comment, it makes more sense. I would stick to PyPi names though and there it isn't available either, right?

Reply:
With that comment, it makes more sense. I would stick to PyPi names though and there it isn't available either, right?

Reply:
I mixed-up the names, there is a `v4-client-py`, but it may be the older package. I just saw that they released a `dydx-v4-client` about 15days ago, and that uses the same git as the one I propose.

It make sense to remove it then since they have the pip package that matches`dydx-v4-client`

Reply:
It is a bit messy, they released a version 1.0.1 in pip, but they refer to it as:
`pip install git+https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-py-v2`, which I translated to their tags `v4-client-js@{{ version }}`. They only tag the JS release

I am going to simplify it and remove the split, simply doing a poetry build and pip install.

I would prefer to follow the github, but I could also use the pypi if you recommend it. I can also contact them, they responded for the JS package